addon/webextension/selector/shooter.js

```diff
@@ -81,8 +77,7 @@ window.shooter = (function () { // eslint-disable-line no-unused-vars
     }).then((url) => {
       const copied = window.clipboard.copy(url);
       return callBackground("openShot", { url, copied });
-    }));
-    exports.deactivate();
+    }).then(() => uicontrol.deactivate()));
```

Comment:
`deactivate` calls `selectorLoader.unloadModules` so it needs to wait until after the background call